CMakeSettings.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "configurations": [
+    {
+      "name": "x64-Debug",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\out\\build\\${name}",
+      "installRoot": "${projectDir}\\out\\install\\${name}",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "-v",
```

Comment:
Is this file really meant to be part of the repo? It seems overly specific and I would prefer not to have it.

Reply:
This is so VS passes the right config to cmake, in particular the vcpkg triplet.